src/core/cpuid.d

```diff
@@ -143,7 +143,7 @@ public:
     /// Is SSE3 supported?
     bool sse3()         {return (miscfeatures&SSE3_BIT)!=0;}
     /// Is SSSE3 supported?
-    bool ssse3()        {return (miscfeatures&SSSE3_BIT)!=0;}
```

Comment:
BTW - why not just make miscfeature immutable and leave functions as is? Is it broken cross-module inlining again? If it's the latter just make them templates...